src/Components/Publishing/Layouts/NewsLayout.tsx

```diff
@@ -9,14 +11,17 @@ interface Props {
   article: ArticleData
   isMobile?: boolean
   isTruncated?: boolean
+  isHovered?: boolean
 }
 
 interface State {
   isTruncated: boolean
+  isHovered: boolean
 }
 
 interface NewsContainerProps {
   isTruncated: boolean
```

Comment:
In Emission we call this state `collapsed`/`expanded` and reserve `truncated` for text, but that's ofc up to you!